config/crd/bases/search.open-cluster-management.io_searches.yaml

```diff
@@ -37,9 +37,9 @@ spec:
             description: SearchSpec defines the desired state of Search
             properties:
               availabilityConfig:
-                description: 'Specifies deployment replication for improved availability.
+                description: '[CURRENTLY UNAVAILABLE] Specifies deployment replication for improved availability.
                   Options are: Basic and High (default)'
-                type: string
+                type: string            
```

Comment:
@Anxhela21 nit we can avoid the new spaces above .

Reply:
done.